ports/directx-dxc/portfile.cmake

```diff
@@ -1,11 +1,7 @@
-set(VCPKG_POLICY_DLLS_IN_STATIC_LIBRARY enabled)
-
 set(DIRECTX_DXC_TAG v1.8.2403)
 set(DIRECTX_DXC_VERSION 2024_03_07)
 
-if (NOT VCPKG_LIBRARY_LINKAGE STREQUAL "dynamic")
-   message(STATUS "Note: ${PORT} always requires dynamic library linkage at runtime.")
-endif()
+vcpkg_check_linkage(ONLY_DYNAMIC_LIBRARY)
```

Comment:
Maybe I didn't fully pay attention to why it was done the original way.
Looking at past PRs, I realize that you deliberately put it the other way because it is independent of vcpkg CRT linkage on Windows and should be blocked on x64-windows-static. `vcpkg_check_linkage(ONLY_DYNAMIC_LIBRARY)` requires the user to customize per-port  library linkage in static triplets to overcome this check.
OTOH the linux libs are dynamically linked to `libc.so.6`, i.e. using dynamic CRT linkage.

Reply:
Put it back then. Thanks!